_organizers/michael-clark.md

```diff
@@ -0,0 +1,10 @@
+---
+github: jefftriplett
+layout: base
+name: Jeff Triplett
+photo_url: /static/img/organizers/jeff.jpg
+role: DEFNA Cofounder and President, Code of Conduct team
+mastodon: https://mastodon.social/@webology
+twitter: webology
+website: https://jefftriplett.com/
```

Comment:
I think you forgot to commit your changes here

Reply:
D'oh!  Just reinstalled Linux so I'm using VSCode rather than my usual PyCharm, which saves changes automatically.

Reply:
Can confirm. Not @jefftriplett 😉

Reply:
But Jeff - I wanna be you!

Reply:
Michael it's even more boring than it seems from afar. More meetings, though. ☕